bucket/youtubedownloader.json

```diff
@@ -0,0 +1,33 @@
+{
+    "version": "1.8.3",
```

Comment:
Why would you add basically dead project, when there are lots of maintained alternatives?

![image](https://user-images.githubusercontent.com/13260377/135091693-78dd2528-98cc-4c5e-80d4-7a73a29bda82.png)

Reply:
Even though this project is in maintenance mode it's not dead yet and there are no alternatives to it right now because it's core is not as same as youtube-dl or any other forks of it.

Reply:
> there are no alternatives to it

Name one feature, which is not covered by other youtube downlaoders already in extras

Reply:
Unlike youtube-dl and its forks YouTube downloads don't get throttled, that's what I've noticed.